src/apm.js

```diff
@@ -36,7 +36,18 @@ module.exports = function (serviceName = name) {
 
   apmConfig = loadConfiguration(process.argv, ROOT_DIR, isKibanaDistributable);
   const conf = apmConfig.getConfig(serviceName);
-  require('elastic-apm-node').start(conf);
+  const apm = require('elastic-apm-node');
+
+  // Filter out username PII
+  apm.addFilter((payload) => {
```

Comment:
Could you add a comment that user data are populated automatically?
What kind of data are considered sensitive? Is email not classified as PII?

Reply:
Anything that can trace back to a specific user is PII. Maybe it would be best to just change all `user` keys to `[REDACTED]`. I didn't test with other security realms that could contain other data besides username

Reply:
I should add that things that are not considered PII in this context include IP address and mac address since they may be used by more than one person and are not necessarily static.

Reply:
>  it would be best to just change all user keys to [REDACTED]

done.